### ISSUE TITLE
Unable to resolve "react-native-view-overflow" from "node_modules/react-native-deck-swiper/Swiper.js"

### ISSUE THREAD

**Issue by JorgeMoralesLopez** [alexbrillant/react-native-deck-swiper#266]
Unable to resolve "react-native-view-overflow" from "node_modules / react-native-deck-swiper / Swiper.js"
Expo sdk 34
"react-native-deck-swiper": "^ 1.5.22"
"react-native-deck-swiper": "^ 1.6.7",
In expo sdk 33 works perfect "react-native-deck-swiper": "^ 1.5.22", no problem with "react-native-view-overflow"
How can I solve this problem?
Thank you!

**Comment by cloudy-ninja** [comment]
got the same issue

**Comment by cloudy-ninja** [comment]
@JorgeMoralesLopez By installing react-native-view-overflow, I have solved this issue.

**Comment by JorgeMoralesLopez** [comment]
Thank you, but I am using expo ... In SDK 33 I had no problems but as of SDK 34 I did.
The case is "react-native-view-overflow" native module is not in Expo, I hope you get a solution.

**Comment by Agungkes** [comment]
@JorgeMoralesLopez  You just need to edit Swiper.js and remove react-native-view-overflow. Because RN version > 0.57 is supports overflow natively.

**Comment by petrovprime** [comment]
Checkout [this PR](https://github.com/alexbrillant/react-native-deck-swiper/pull/269) I just created to resolve this issue.

**Comment by jimmysafe** [comment]
I have the same issue.. RNViewOverflow was not found in the UIManager,

Any update on this issue?

**Comment by JorgeMoralesLopez** [comment]
I updated to 1.6.7 and made the changes to make it work in expo and the problem of view overflow no longer appears but the swiper has a strange effect on me when the last 4 cards remain. I also had the index in my database and tb does strange things. Create a function to restart the deck and it doesn't work as it should.
I really don't know why this happens ...

**Comment by Baksha2008** [comment]
yarn add react-native-view-overflow ???

**Comment by yashkapoor8710** [comment]
Yes we have to use yarn add react-native-view-overflow after installing the react-native-deck-swiper
